oms.sh

```diff
@@ -1,75 +1,25 @@
 #!/bin/bash
 
-echo -e "USAGE: bash ./oms.sh [--no-up | --build | [--down | --down_v] | --push]\n";
+## This file gathers all docker-compose.yml files from [subfolders]/docker/docker-compose.yml and merges them with the -f option of the compose command.
+## Instead of docker-compose up -d, write ./oms.sh up -d
 
-# Default settings
-UP=YES
-BUILD=NO
-DOWN=NO
-DOWN_V=NO
-PUSH=NO
 
-#Parse input
-for i in "$@"
-do
-case $i in
-    --no-up)
-    UP=NO
-    shift # past argument=value
-    ;;
-    --build)
-    BUILD=YES
-    shift # past argument=value
-    ;;
-    --down)
-    DOWN=YES
-    shift # past argument=value
-    ;;
-    --down_v)
-    DOWN_V=YES
-    shift # past argument=value
-    ;;
-    --push)
-    PUSH=YES
-    shift # past argument=value
-    ;;
-    *)
-            # unknown option
-    ;;
-esac
-done
-
-#Display input
-echo -e "UP            (--no-up)       = ${UP}"
-echo -e "BUILD         (--build)       = ${BUILD}"
-echo -e "DOWN          (--down)        = ${DOWN}"
-echo -e "DOWN_V        (--down_v)      = ${DOWN_V}"
-echo -e "PUSH          (--push)        = ${PUSH}"
-
-
-#
-## Script
-#
+if ! [[ -f ".env" ]]; then
+    echo -e "[OMS] Copying .env file from .env.example"
+    cp ./.env.example ./.env
+fi
 
 
-echo -e "OMS: Create network..."
-docker network create OMS
-
 ## Declare docker-compose.yml folders
-declare -a arr=("docker" "oms-core/docker")
-
-## Save base path
-BASE_PATH=$(pwd)
+declare -a services=("oms-global" "oms-core")
```

Comment:
What about instead of declaring existing services just go through all subfolders and if it has a docker-compose.yml it will be included? This way there is 1 step less in adding a new service.